### PR TITLE
liquibase scripts added

### PR DESCRIPTION
**Pull request by GeorgeT88**

### CODE REVIEW

File: src/main/resources/db/changelog/v1.0.0/2-create-cart_entry-table.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="product_name" type="varchar(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="quantity" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cart_id" type="bigint">
+                <constraints nullable="false"/>
```
Comment (florinv23) [inline src/main/resources/db/changelog/v1.0.0/2-create-cart_entry-table.xml:27]: we need a foreign constraint to cart id column

Comment (GeorgeT88) [reply src/main/resources/db/changelog/v1.0.0/2-create-cart_entry-table.xml:27]: Done! good remark! 👍

Comment (GeorgeT88) [reply src/main/resources/db/changelog/v1.0.0/2-create-cart_entry-table.xml:27]: added this change to Order aswell

**Review by florinv23 — APPROVED** [review]
the changes look good

**Pull request merged**